cg/cli/demultiplex/demux.py

```diff
@@ -179,76 +187,5 @@ def copy_novaseqx_flow_cells(context: CGConfig):
     for flow_cell in flow_cells.iterdir():

```

Comment:
```suggestion
    for flow_cell_directory in flow_cell_directories.iterdir():

```

Reply:
Clear by context (flow_cells is a Path and we are using `iter_dir`)